lib/engine/game/g_18_usa/step/scrap_train_module.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18USA
+      module Step
+        module ScrapTrainModule
+          def scrappable_trains(entity)
```

Comment:
The module and step both do this. I recommend putting most of the logic in the module (so it can be used by the buy_sell_par_shares step) and keep the scrap_trains step light.

Reply:
Done in an earlier round of cleanup today